Makefile

```diff
@@ -20,5 +20,5 @@ postgres:
 pygments:
 	out=$$(edb gen-meta-grammars edgeql) && \
 		echo "$$out" > edb/edgeql/pygments/meta.py
-	out=$$(edb gen-meta-grammars edgeql eschema) && \
+	out=$$(edb gen-meta-grammars edgeql) && \
```

Comment:
We don't need eschema/pygments anymore

Reply:
Pygments dropped from code and make target.